src/main/java/com/lunark/lunark/properties/model/Property.java

```diff
@@ -77,6 +72,21 @@ public void setApproved(boolean approved) {
     @Transient
     private Clock clock = Clock.systemDefaultZone();
     private PropertyType type;
+    @ManyToOne
+    @JoinColumn(name = "host_id")
+    private Account host;
```

Comment:
This could be the reason why the property test is failing. This field is now added to the constructor by the @AllArgsConstructor, but the host is not provided in the outdated tests.

Reply:
I have fixed the problem.